tests/compose.yml

```diff
@@ -0,0 +1,78 @@
+name: vault-manager-test
+
+services:
+
+  keycloak:
+    image: quay.io/keycloak/keycloak
+    ports:
+      - "8180:8180"
+    environment:
+      KEYCLOAK_USER: admin
+      KEYCLOAK_PASSWORD: admin
+    command: start-dev --http-port 8180 --http-relative-path /auth
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://127.0.0.1:8180/auth"]
+      interval: 10s
+      timeout: 5s
+      retries: 12
+      start_period: 10s
+
+  keycloak-cli:
+    image: quay.io/app-sre/keycloak-config-cli:5.0.0-17.0.1
+    environment:
+      - KEYCLOAK_URL=http://keycloak:8180/auth
+      - KEYCLOAK_USER=${KEYCLOAK_USER}
+      - KEYCLOAK_PASSWORD=${KEYCLOAK_PASSWORD}
+      - KEYCLOAK_AVAILABILITYCHECK_ENABLED=true
+      - IMPORT_FILES=/config/*
+    volumes:
+      - ../tests/keycloak:/config:Z
+    depends_on:
+      - keycloak
+
+  qontract-server:
+    image: quay.io/app-sre/qontract-server:669e975
+    environment:
+      LOAD_METHOD: fs
+      DATAFILES_FILE: /bundle/data.json
+    volumes:
+      - ../tests/app-interface:/bundle:Z
+
+  primary-vault:
+    image: quay.io/app-sre/vault:1.17.1
```

Comment:
can we move to the redhat vault image for these?

Reply:
Done.